lte/gateway/c/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -325,8 +325,7 @@ void clean_stale_enb_state(
   enb_description_t* stale_enb_association = NULL;
 
   hashtable_ts_apply_callback_on_elements(
-      (hash_table_ts_t* const) & state->enbs,
-      get_stale_enb_connection_with_enb_id, new_enb_association,
+      &state->enbs, get_stale_enb_connection_with_enb_id, new_enb_association,
```

Comment:
was the casting creating an issue?

Reply:
The casting was not creating any issue. It is not necessary here, and clang-format was separating the address operand from the variable name, so cleaned it up for better readability.